src/libmongoc/tests/test-mongoc-async.c

```diff
@@ -235,6 +235,7 @@ test_large_ismaster_helper (mongoc_async_cmd_t *acmd,
    }
    ASSERT_CMPINT (result, ==, MONGOC_ASYNC_CMD_SUCCESS);
 
+   ASSERT_HAS_FIELD(bson, "ismaster");
```

Comment:
Leaving this here since previously we didn't print the BSON object if the line below failed.